File: app/src/main/java/com/example/massa/luxvilla/sugestoes/SearchSugestionsProvider.java
package com.example.massa.luxvilla.sugestoes;

import android.content.SearchRecentSuggestionsProvider;

/**
 * Created by massa on 07/04/2016.
 */
public class SearchSugestionsProvider extends SearchRecentSuggestionsProvider {
    public static final String AUTHORITY="com.example.massa.luxvilla.sugestoes.SearchSugestionsProvider";
    public static final int MODE=DATABASE_MODE_QUERIES;


    public SearchSugestionsProvider(){
        setupSuggestions(AUTHORITY , MODE);
    }
}

File: app/src/main/java/com/example/massa/luxvilla/sugestoes/SearchSugestionsProvider.kt
package com.example.massa.luxvilla.sugestoes

import android.content.SearchRecentSuggestionsProvider

/**
 * Created by massa on 27/10/2017.
 */
class SearchSugestionsProvider : SearchRecentSuggestionsProvider() {
    init {
        setupSuggestions(AUTHORITY, MODE)
    }

    companion object {
        val AUTHORITY = "com.example.massa.luxvilla.sugestoes.SearchSugestionsProvider"
        val MODE = SearchRecentSuggestionsProvider.DATABASE_MODE_QUERIES
    }
}
